pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -973,4 +973,13 @@ self: super: {
   amqp-utils = super.amqp-utils.override {
     amqp = dontCheck super.amqp_0_18_1;
   };
+
+  # depends on haskell-gi-overloading-0.0
+  ltk = doJailbreak super.ltk;
+
+  # tests fail
+  leksah-server = dontCheck super.leksah-server;
```

Comment:
I very much doubt that leksah is going work after this. The test suite fails because leksah has broken assumptions about the underlying Cabal library. Those won't magically fix themselves by disabling the test suite. I would REALLY recommend that you report this issue upstream.

Reply:
I opened https://github.com/leksah/leksah-server/issues/15. Also tried https://github.com/leksah/leksah-server/commit/c9bec9012db4cde04cb47e13849010a1d2d8f85e, but output still looks the same after applying this patch.